src/main/java/frc/core/util/sysId/DrivetrainSysIdTunning.java

```diff
@@ -0,0 +1,140 @@
+package frc.core.util.sysId;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+
+import static edu.wpi.first.units.MutableMeasure.mutable;
+import static edu.wpi.first.units.Units.Meters;
+import static edu.wpi.first.units.Units.MetersPerSecond;
+import static edu.wpi.first.units.Units.Volts;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class DrivetrainSysIdTunning implements Subsystem {
```

Comment:
you can remove "implements Subsystem" because it is not a subsystem, it just gets the drivetrain subsystem variables and make some code

Reply:
falei em ingles sem querer aqui, mas remove o implements Subsystem